overlord/configstate/configcore/experimental_test.go

```diff
@@ -0,0 +1,83 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package configcore_test
+
+import (
+	"fmt"
+	"path/filepath"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/overlord/configstate/configcore"
+	"github.com/snapcore/snapd/testutil"
+)
+
+type experimentalSuite struct {
+	configcoreSuite
+	features []string
+}
+
+var _ = Suite(&experimentalSuite{
+	features: []string{
```

Comment:
Should we just get this from "experimental.go" via export_test.go and the experimentalFlags that are defined there? As it is right now it misses the new `per-user-mount-namespace` as part of th test.

Reply:
I would like to do a follow-up and use the features/ package as a canonical defining set, does that sound sensible?

Reply:
yeah